core-feature-pack/src/main/resources/modules/system/layers/base/sun/jdk/main/module.xml

```diff
@@ -52,8 +52,145 @@
                 <path name="com/sun/jndi/url/rmi"/>
                 <path name="com/sun/media/sound"/>
                 <path name="com/sun/crypto/provider"/>
-                <path name="com/sun/org/apache/bcel/internal/classfile"/>
+                <path name="com/sun/org/apache/xml/internal/security/algorithms"/>
```

Comment:
via JAXP factories are resolved and loaded from jdk, and if they are not included in this file they wont work.
I just added all com/sun/org/apache/xml paths here.